### ISSUE TITLE
Reformat CHANGELOG.md

### ISSUE THREAD

**Issue by InsanusMokrassar** [InsanusMokrassar/TelegramBotAPI#147]
Currently `CHANGELOG.md` have next format:

```markdown
## MAJOR VERSION

Some notes

### LAST MINOR VERSION OF MAJOR VERSION

Some notes

### FIRST MINOR VERSION OF MAJOR VERSION

Some notes
```

As for this issues this format should be changed to the next one:

```markdown
## LAST MINOR VERSION OF MAJOR VERSION

Some notes

## FIRST MINOR VERSION OF MAJOR VERSION

Some notes

## MAJOR VERSION

Some notes
```

**Comment by leobia** [comment]
Hi! I would love to help you on this issue.

If i did get it right you want something like this:

from:

```
## 0.28.0
...
### 0.28.4
...
### 0.28.3
...
### 0.28.2
...
### 0.28.1
...

```

to:


```
## 0.28.4
...
## 0.28.3
...
## 0.28.2
...
## 0.28.1
...
## 0.28.0
...

```

Right?

**Comment by InsanusMokrassar** [comment]
Solved in #148
